examples/septentrio_example.cc

```diff
@@ -76,26 +104,34 @@ int main(int argc, char *argv[], char *envp[]) {
   point_one::polaris::PolarisAsioClient polaris_client(
       io_loop, FLAGS_polaris_api_key, "septentrio12345", settings);
 
-  // This callback will forward RTCM correction bytes received from the server
-  // to the septentrio.
+  // This callback will forward RTCM correction bytes received from Polaris to
+  // the Septentrio, and to an output UDP port if enabled.
```

Comment:
comma splice

Reply:
Ok. In that case might go with colon since that's more typical for URLs.